src/core/sim.py

```diff
@@ -73,13 +72,24 @@ def should_stop(self) -> bool:
             log.debug(f"{self.state}")
             return True
 
-        if self.num_iters > 1e5:
+        if self.num_iters > 1e6:
```

Comment:
This will have to become a sim parameter so we don't have to change this in the future.

Reply:
yes! Good call